core/src/style/pointStyle.cpp

```diff
@@ -15,6 +15,8 @@
 #include "data/propertyItem.h" // Include wherever Properties is used!
 #include "scene/stops.h"
 
+constexpr float texture_scale = static_cast<float>(std::numeric_limits<int16_t>::max());
```

Comment:
Yup, I just don't like magic numbers. But this is fine with me. My initial comment was to get rid of magic number :D.